CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## Changelog
 
+### v1.1.1 (2024-11-11)
```

Comment:
Add date of (just before) the PyPI release.

Reply:
Yes, I will release v1.1.1 today with the hotfixes.